### PR TITLE
Changes, arranges stakced on staging to master

### PR DESCRIPTION
**Pull request by hanbyul-here**
- There has been changes going on staging that don't need to sit on staging. my bad. pushing them to master

### CODE REVIEW

File: src/stylesheets/common/_variables.scss [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 // Display type used for headings, logo, etc.
 $display-font-family: 'Poppins', 'Open Sans', Helvetica, Arial, sans-serif;
-$display-font-weight-bold: 600;
+$display-font-weight-bold: 700;
```
Comment (rfriberg) [inline src/stylesheets/common/_variables.scss:20]: I think this is supposed to stay at 600 for Poppins.

Comment (hanbyul-here) [reply src/stylesheets/common/_variables.scss:20]: oh @souperneon told me to change it to 700. but probably not ready to go to master yet. how do you think @souperneon ?

Comment (rfriberg) [reply src/stylesheets/common/_variables.scss:20]: Oh, as long as it was intentional.  I was worried it was a holdover from all the previous font changes.

**Review by rfriberg — APPROVED** [review]

File: src/stylesheets/common/_variables.scss (hunk 1 above)
Comment (souperneon) [reply src/stylesheets/common/_variables.scss:20]: Yes, 700 please.

**Pull request merged**